Makefile

```
@@ -93,11 +93,19 @@ mod:
 	$(GO) mod tidy
 	$(GO) mod download
 
-## deploy: Deploy to /opt/dbkrab (build, install, restart)
+## deploy: Deploy to /opt/dbkrab (build, install, restart via systemd)
 deploy:
```

checkmake (0.3.2)

[warning] Target body for "deploy" exceeds allowed length of 5 lines (11).

(maxbodylength)